src/werkzeug/debug/shared/debugger.js

```diff
@@ -1,3 +1,25 @@
+function fadeOut(el){
+    el.style.opacity = 1;
+
+    (function fade() {
+        if ((el.style.opacity -= .1) < 0) el.style.display = "none";
```

Comment:
Use brackets around the if's and else's. Makes it less error-prone.

Reply:
Does `(el.style.opacity - .1 < 0)` work instead of `((el.style.opacity -= .1) < 0)` ?

Reply:
If `el.style.opacity` needs to be set, I'd prefer doing it outside of the boolean expression.
```
el.style.opacity -= .1;
if (el.style.opacity < 0) {
 ...
} else {
 ...
}
```